tests/Git/Concerns/PreparesTempRepos.php

```diff
@@ -38,7 +38,7 @@ protected function createTempRepo($path)
 
         $process = Process::create($path);
 
-        $process->run('git init');
+        $process->run('git init -b master');
```

Comment:
Not that it really matters because it's a test, but I'm just curious why you need to force the branch name?

Reply:
I mentioned in commit description, but this was being logged to ray...

```
Process Command: git init Error: hint: Using 'master' as the name for the initial branch. This default branch name hint: is subject to change. To configure the initial branch name to use in all hint: of your new repositories, which will suppress this warning, call: hint: hint: git config --global init.defaultBranch hint: hint: Names commonly chosen instead of 'master' are 'main', 'trunk' and hint: 'development'. The just-created branch can be renamed via this command: hint: hint: git branch -m
```

Being explicit with branch name in that testing trait just fixes that error is all 👍